linkis-commons/linkis-rpc/src/main/scala/org/apache/linkis/rpc/RPCReceiveRestful.scala

```diff
@@ -152,29 +152,36 @@ private[rpc] class RPCReceiveRestful extends RPCReceiveRemote with Logging {
     val obj = RPCConsumer.getRPCConsumer.toObject(message)
     val event = RPCMessageEvent(obj, BaseRPCSender.getInstanceInfo(message.getData))
     rpcReceiverListenerBus.post(event)
+    toMessage(Unit)
   }
 
-  private def receiveAndReply(
+  private def receiveAndReplyWithMessage(
       message: Message,
-      opEvent: (Receiver, Any, Sender) => Message
+      opEvent: (Receiver, Any, Sender) => Any
   ): Message = catchIt {
     val obj = RPCConsumer.getRPCConsumer.toObject(message)
     val serviceInstance = BaseRPCSender.getInstanceInfo(message.getData)
     val event = RPCMessageEvent(obj, serviceInstance)
-    event.map(opEvent(_, obj, event)).getOrElse(RPCProduct.getRPCProduct.notFound())
+    event
+      .map(receiver => {
+        logger.debug(s"show the receiver ${receiver.getClass}")
```

Comment:
should to add if (logger.isDebugEnabled())

Reply:
i will change the string format to string template, to avoid extra operator